### PR TITLE
Fix an incorrect wording "scale out or in a node"

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Fix an incorrect wording "scale out/in a node".

+ scale out a node --> add a node/scale out a cluster
+ scale in a node --> remove a node/take a node offline/scale in a cluster

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: tiflash/troubleshoot-tiflash.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The issue might occur due to different reasons. It is recommended that you troub
      ulimit -n 1000000
      ```
 
-3. Use the PD Control tool to check whether there is any TiFlash instance that failed to go offline on the node (same IP and Port) and force the instance(s) to go offline. For detailed steps, refer to [Scale in a TiFlash node](/scale-tidb-using-tiup.md#scale-in-a-tiflash-node).
+3. Use the PD Control tool to check whether there is any TiFlash instance that failed to go offline on the node (same IP and Port) and force the instance(s) to go offline. For detailed steps, refer to [Scale in a TiFlash cluster](/scale-tidb-using-tiup.md#scale-in-a-tiflash-cluster).
```
Comment (TomShawn) [inline tiflash/troubleshoot-tiflash.md:33]:
```suggestion
3. Use the PD Control tool to check whether there is any TiFlash instance that failed to go offline on the node (same IP and Port) and force the instance(s) to go offline. For detailed steps, refer to [Scale in a TiFlash cluster](/scale-tidb-using-tiup.md##scale-in-a-tiflash-cluster).
```

File: tiflash/troubleshoot-tiflash.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The issue might occur due to different reasons. It is recommended that you troub
      ulimit -n 1000000
      ```
 
-3. Use the PD Control tool to check whether there is any TiFlash instance that failed to go offline on the node (same IP and Port) and force the instance(s) to go offline. For detailed steps, refer to [Scale in a TiFlash node](/scale-tidb-using-tiup.md#scale-in-a-tiflash-node).
+3. Use the PD Control tool to check whether there is any TiFlash instance that failed to go offline on the node (same IP and Port) and force the instance(s) to go offline. For detailed steps, refer to [Scale in a TiFlash cluster](/scale-tidb-using-tiup.md##scale-in-a-tiflash-cluster).
```
Comment (TomShawn) [inline tiflash/troubleshoot-tiflash.md:33]:
```suggestion
3. Use the PD Control tool to check whether there is any TiFlash instance that failed to go offline on the node (same IP and Port) and force the instance(s) to go offline. For detailed steps, refer to [Scale in a TiFlash cluster](/scale-tidb-using-tiup.md#scale-in-a-tiflash-cluster).
```

File: experimental-features-4.0.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ This document introduces the experimental features of TiDB v4.0. It is **NOT** r
...
-+ Elastic scheduling is an experimental feature based on Kubernetes, which enables TiDB to dynamically scale in and out nodes. This feature can effectively mitigate the high workload during peak hours of an application and saves unnecessary overhead. See [Enable TidbCluster Auto-scaling](https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tidb-cluster-auto-scaling) for details.
++ Elastic scheduling is an experimental feature based on Kubernetes, which enables TiDB to dynamically add and remove nodes. This feature can effectively mitigate the high workload during peak hours of an application and saves unnecessary overhead. See [Enable TidbCluster Auto-scaling](https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tidb-cluster-auto-scaling) for details.
```
Comment (lilin90) [inline experimental-features-4.0.md:13]: Suggest keeping the word "scale".
```suggestion
+ Elastic scheduling is an experimental feature based on Kubernetes, which enables TiDB to dynamically scale out and scale in clusters. This feature can effectively mitigate the high workload during peak hours of an application and saves unnecessary overhead. See [Enable TidbCluster Auto-scaling](https://docs.pingcap.com/tidb-in-kubernetes/stable/enable-tidb-cluster-auto-scaling) for details.
```

File: scale-tidb-using-tiup.md [hunk 4]
```diff
@@ -319,11 +319,11 @@ Before the node goes down, make sure that the number of remaining nodes in the T
 
 2. Wait for the TiFlash replicas of the related tables to be deleted. [Check the table replication progress](/tiflash/use-tiflash.md#check-the-replication-progress) and the replicas are deleted if the replication information of the related tables is not found.
 
-### 2. Scale in the TiFlash node
+### 2. Remove a TiFlash node
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:322]:
```suggestion
### 2. Perform the scale-in operation
```

File: scale-tidb-using-tiup.md [hunk 5]
```diff
@@ -333,17 +333,17 @@ Next, perform the scale-in operation with one of the following solutions.
...
+2. Remove the TiFlash node (assume that the node name is `10.0.1.4:9000` from Step 1):
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     tiup cluster scale-in <cluster-name> --node 10.0.1.4:9000
     ```
 
-#### Solution 2: Manually scale in the TiFlash node
+#### Solution 2: Manually remove the TiFlash node
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:344]:
```suggestion
#### Solution 2: Manually remove a TiFlash node
```

File: scale-tidb-using-tiup.md [hunk 6]
```diff
@@ -319,11 +319,11 @@ Before the node goes down, make sure that the number of remaining nodes in the T
 
 2. Wait for the TiFlash replicas of the related tables to be deleted. [Check the table replication progress](/tiflash/use-tiflash.md#check-the-replication-progress) and the replicas are deleted if the replication information of the related tables is not found.
 
-### 2. Scale in the TiFlash node
+### 2. Remove a TiFlash node
 
 Next, perform the scale-in operation with one of the following solutions.
 
-#### Solution 1: Using TiUP to scale in the TiFlash node
+#### Solution 1: Using TiUP to remove a TiFlash node
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:326]:
```suggestion
#### Solution 1: Use TiUP to remove a TiFlash node
```

File: scale-tidb-using-tiup.md [hunk 7]
```diff
@@ -22,7 +22,7 @@ For example, if the original topology of the cluster is as follows:
 | 10.0.1.1 | TiKV |
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:22]: Please also update line 11.

File: scale-tidb-using-tiup.md [hunk 8]
```diff
@@ -436,9 +436,9 @@ The steps to manually clean up the replication rules in PD are below:
     curl -v -X DELETE http://<pd_ip>:<pd_port>/pd/api/v1/config/rule/tiflash/table-45-r
     ```
 
-## Scale in a TiCDC node
+## Scale in a TiCDC cluster
 
-If you want to remove the TiCDC node from the `10.0.1.4` host, take the following steps.
+If you want to remove a TiCDC node from the `10.0.1.4` host, take the following steps.
```
Comment (lilin90) [inline scale-tidb-using-tiup.md:441]: It should be `the` in this sentence.
```suggestion
If you want to remove the TiCDC node from the `10.0.1.4` host, take the following steps:
```

File: scale-tidb-using-tiup.md (hunk 7 above)
Comment (TomShawn) [reply scale-tidb-using-tiup.md:22]: Done in https://github.com/pingcap/docs/pull/3713/commits/64ca17f1bd695d6d99e0bec0e1e7405dcedce8be

**Review by lilin90 — APPROVED** [review]
LGTM

**Pull request merged**